projects/commudle-admin/src/app/feature-modules/public-community/components/speakers/speakers.component.scss

```diff
@@ -0,0 +1,73 @@
+@import "/projects/commudle-admin/src/assets/styles/variables.scss";
+
+.speakers-container {
+  nb-card {
+    background: transparent;
+    border: none;
+    nb-card-body {
+      .user-card {
+        position: relative;
+        width: 200px;
+        height: 280px;
+        background: $white;
+        transition: all 0.5s ease;
+        box-shadow: 0 3px 6px rgba(000, 000, 0, 0.3);
+        border-radius: 5px;
+        overflow: hidden;
+
+        &:hover {
+          transform: translateY(-5px);
+          box-shadow: 0 5px 15px rgba(000, 000, 0, 0.5);
+        }
+
+        .banner {
+          position: absolute;
+          width: 200px;
+          height: 80px;
+          background: linear-gradient(#536effff, #29a5ffff);
+          img {
+            width: 100%;
+            height: 100%;
```

Comment:
We must not do 100% height and width together because it will distort the aspect ratio (we discussed it earlier also)
Please let's just keep width 100% on